scripts/android/cmake.sh

```diff
@@ -55,6 +55,16 @@ while [[ $# -gt 0 ]]; do
     RELEASE=
     shift
     ;;
+  --sentrydsn)
+    SENTRY_DSN="$2"
```

Comment:
What does DSN stand for?

Reply:
i do not know, sentry calls that value that :D

Reply:
LOL SHIP IT

Reply:
(It's Data Source Name [apparently](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjylOKjv8T7AhWQSPEDHSUBCzYQFnoECBIQAQ&url=https%3A%2F%2Fdocs.sentry.io%2Fproduct%2Fsentry-basics%2Fdsn-explainer%2F&usg=AOvVaw2dtdfVxf8bL3rzFraKUCR6))